lib/App/Config/Chronicle.pm

```diff
@@ -358,12 +361,19 @@ sub _save_dynamic {
     $settings->{_rev}   = Time::HiRes::time();
     $self->chronicle_writer->set($self->setting_namespace, $self->setting_name, $settings, Date::Utility->new);
 
+    # since we now have the most recent data, we better set the
+    # local version as well.
+    $self->data_set->{version} = $settings->{_rev};
+    $self->_updated_at($settings->{_rev});
+
     return 1;
 }
 
 =head2 current_revision
 
-loads setting from chronicle reader and returns the last revision and drops them
+Loads setting from chronicle reader and returns the last revision
+
+It is more likely that you want C<loaded_revision> in regular use
```

Comment:
```suggestion
It is more likely that you want L</loaded_revision> in regular use
```

Reply:
Committed suggested change.  After all these years I still really suck at POD and its conventions.